### PR TITLE
Update example card

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
## What kind of change does this PR introduce?

Update example card to match current UI.

## What is the current behavior?

CleanShot 2023-10-04 at 22 05 29@2x

## What is the new behavior?

CleanShot 2023-10-04 at 22 05 33@2x

## Additional context

Didn't know if I should keep or get rid of the color separation or the creator image/name.

### CODE REVIEW

File: apps/www/components/ExampleCard.tsx [hunk 1]
```diff
@@ -1,84 +1,44 @@
...
-            <span className="ml-1 inline-block">
-              <IconGitHub size={14} />
-            </span>
+        <div className="flex items-start gap-2">
+          <Link href={props.repo_url}>
+            <a target="_blank" tabIndex={-1}>
+              <Button size="tiny" type="default" iconRight={<IconArrowUpRight />}>
+                View Code
+              </Button>
+            </a>
```
Comment (fsansalvadore) [inline apps/www/components/ExampleCard.tsx:21]: With next13 now next/link `<Link>` component is self contained, you don't need the inner `<a>` tag.
So you can remove the <a> tag. Also, the <Button> component should be outside using `asChild` prop.

Like this:
```tsx
<Button size="tiny" type="default" iconRight={<IconArrowUpRight />} asChild>
  <Link href={props.repo_url} target="_blank">
    View Code
  </Link>
</Button>
```

File: apps/www/components/ExampleCard.tsx [hunk 2]
```diff
@@ -1,84 +1,44 @@
...
-            {props.vercel_deploy_url && (
-              <a target="_blank" href={props.vercel_deploy_url}>
+          {props.demo_url && (
+            <Link href={props.demo_url}>
+              <a target="_blank" tabIndex={-1}>
+                <Button size="tiny" type="default" iconRight={<IconArrowUpRight />}>
+                  Launch Demo
+                </Button>
+              </a>
+            </Link>
```
Comment (fsansalvadore) [inline apps/www/components/ExampleCard.tsx:30]: Same as above

File: apps/www/components/ExampleCard.tsx [hunk 3]
```diff
@@ -1,84 +1,44 @@
...
+          )}
+          {props.vercel_deploy_url && (
+            <Link href={props.vercel_deploy_url}>
+              <a target="_blank">
```
Comment (fsansalvadore) [inline apps/www/components/ExampleCard.tsx:34]: Same here, you can remove the <a> tag and move `target` to the Link tag.

File: apps/www/components/ExampleCard.tsx [hunk 4]
```diff
@@ -1,84 +1,44 @@
...
-                objectFit="cover"
-              />
-            </div>
-            <span className="text-foreground ml-2 text-sm">{props.author}</span>
+    <div className="relative rounded-xl bg-scale-400 from-scale-800 to-scale-800 p-px transition-all shadow-md h-full">
+      <div className="relative h-full z-10 rounded-xl bg-scale-200 dark:bg-scale-300 overflow-hidden transition-all text-scale-1100 flex flex-col p-5">
+        <div className="mb-4 lg:mb-8 flex-1">
+          <h2 className="text text-lg font-medium">{props.title}</h2>
+          <div className="my-2 block">
+            <p className="text-light">{props.description}</p>
```
Comment (fsansalvadore) [inline apps/www/components/ExampleCard.tsx:12]:
```suggestion
            <p className="text-sm">{props.description}</p>
```

File: apps/www/components/ExampleCard.tsx [hunk 5]
```diff
@@ -1,84 +1,44 @@
...
-                alt={props.author + ' GitHub profile picture'}
-                layout="fill"
-                objectFit="cover"
-              />
-            </div>
-            <span className="text-foreground ml-2 text-sm">{props.author}</span>
+    <div className="relative rounded-xl bg-scale-400 from-scale-800 to-scale-800 p-px transition-all shadow-md h-full">
+      <div className="relative h-full z-10 rounded-xl bg-scale-200 dark:bg-scale-300 overflow-hidden transition-all text-scale-1100 flex flex-col p-5">
+        <div className="mb-4 lg:mb-8 flex-1">
+          <h2 className="text text-lg font-medium">{props.title}</h2>
```
Comment (fsansalvadore) [inline apps/www/components/ExampleCard.tsx:10]:
```suggestion
          <h2 className="text text-sm font-medium">{props.title}</h2>
```

**Review by fsansalvadore — CHANGES_REQUESTED** [review]

**Comment by Isaiah-Hamilton** [comment]
Closing because the component is no longer being used.

**Pull request closed without merging**